jumanji/environments/packing/bin_pack/env.py

```diff
@@ -739,6 +743,148 @@ def _get_intersections_dict(
                 intersections_mask_dict[direction] &= ~to_remove
         return intersections_ems_dict, intersections_mask_dict
 
+    def merge_same_height_ems(self, state: State) -> None:
+        def compute_merge_mask(args: Tuple) -> Tuple:
+            def compute_vector_difference(a: chex.Array, b: chex.Array) -> chex.Array:
+                return jnp.isclose(
+                    jnp.reshape(a, (len(a), 1))
```

Comment:
I think jnp.expand_dims(a,-1) would be clearer than using both reshape and len here?

Reply:
A small docstring explaining the purpose/what this is doing would be best

Reply:
done ✅